razzle.config.js

```diff
@@ -29,8 +29,8 @@ module.exports = {
     // This is to override bundle performance test
     if (process.env.CI) {
       appConfig.performance = {
-        maxAssetSize: 400000,
-        maxEntrypointSize: 400000,
+        maxAssetSize: 410000,
```

Comment:
We shouldn't keep arbitrarily increasing the bundle size. This work should be blocked until https://github.com/bbc/simorgh/issues/466 can be merged, and then this code removed.

Reply:
Ok. Yeah it is a pain.

Reply:
Unblocked by https://github.com/bbc/simorgh/pull/469.